5-WebApp-AuthZ/5-2-Groups/Startup.cs

```diff
@@ -36,16 +36,13 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             // Sign-in users with the Microsoft identity platform
-            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
-                .AddSignIn("AzureAD", Configuration, options =>
+            services.AddSignIn(options =>
                 {
-                    Configuration.Bind("AzureAD", options);
-
                     // Uncomment the following lines code instruct the asp.net core middleware to use the data in the "groups" claim in the [Authorize] attribute and for User.IsInrole()
                     // See https://docs.microsoft.com/en-us/aspnet/core/security/authorization/roles for more info.
                     //    // Use the groups claim for populating roles
                     //    options.TokenValidationParameters.RoleClaimType = "groups";
-                });
+                }, options => { });
```

Comment:
the second options should probably do the binding as well? `Configuration.Bind("AzureAD", options);` ?